### ISSUE TITLE
[BUG] Ethash-Lowmemory mining on Prohashing fails

### ISSUE THREAD

**Issue by RainbowMiner** [sp-hash/TeamBlackMiner#306]
```
TBMiner.exe --algo ethash --hostname eu.prohashing.com --port 3358 --wallet xyz --worker-name abc --server-passwd a=ethash-lowmemory,n=abc --cuda-devices [0]
```

.. leads to a pool error message:
```
00:00:00 [2022-09-20 12:16:05.135]
00:00:00 [2022-09-20 12:16:05.275] OpenCL driver version: 516.40
00:00:00 [2022-09-20 12:16:05.307] Cuda driver version: 11.7
00:00:00 [2022-09-20 12:16:05.307] Cuda runtime API version: 11.5
00:00:00 [2022-09-20 12:16:05.307]
00:00:00 [2022-09-20 12:16:05.307]
00:00:00 [2022-09-20 12:16:05.307]
00:00:00 [2022-09-20 12:16:05.307] Welcome to Team Black Miner 1.67
00:00:00 [2022-09-20 12:16:05.307]
00:00:00 [2022-09-20 12:16:05.307]
00:00:00 [2022-09-20 12:16:06.072] GPU0 NVIDIA GeForce GTX 1070 8GB added to miner as Cuda device
00:00:00 [2022-09-20 12:16:06.088] Trying eu.prohashing.com:3358 (ethash)
00:00:00 [2022-09-20 12:16:06.478] Pool responded successfully to subscribe
00:00:00 [2022-09-20 12:16:06.478] Difficulty is set to 0.250000
00:00:00 [2022-09-20 12:16:06.666] mining.extranonce.subscribe: Pool reponded with error
00:00:00 [2022-09-20 12:16:06.666] Pool responded successfully to authorize
00:00:00 [2022-09-20 12:16:06.682] mining.authorize: Pool reponded with error
00:00:00 [2022-09-20 12:16:06.682] Error(-2): Authorization failed because there are no coins available to mine for algorithm 10.
00:00:00 [2022-09-20 12:16:06.682] Shutting down threads
00:00:00 [2022-09-20 12:16:06.697] Stratum thread exited
```

Running any other Ethash miner (Trex, Ethminer, Gminer, ...) with the above configuration works.
Mining Etchash with TBminer on Prohashing works

**Comment by UselessGuru** [comment]
I have also reported this issue with ProHashing support.
It seems to be a general problem with EthashLowMem algo. Other miners (PhoenixMiner, TTMiner & NanoMiner also fail)

**Comment by sp-hash** [comment]
fixed in v1.69
